host/class/cdc/usb_host_cdc_acm/test_app/pytest_usb_host_cdc.py

```diff
@@ -14,14 +14,18 @@
 @pytest.mark.parametrize('count', [
     2,
 ], indirect=True)
-def test_usb_host(dut: Tuple[IdfDut, IdfDut]) -> None:
+def test_usb_host_cdc(dut: Tuple[IdfDut, IdfDut]) -> None:
     device = dut[0]
     host = dut[1]
 
-    # 1.1 Prepare USB device for CDC test
+    # 1 Prepare USB device for CDC test
     device.expect_exact('Press ENTER to see the list of tests.')
     device.write('[cdc_acm_device]')
     device.expect_exact('USB initialization DONE')
 
-    # 1.2 Run CDC test
-    host.run_all_single_board_cases(group='cdc_acm')
+    # 2 Run CDC test
+    if host.target == 'esp32p4':
+        # rx_buffer test can't be run for esp32p4, reason: RX buffer append is not yet supported on ESP32-P4
+        host.run_all_single_board_cases(group='hs_host')
```

Comment:
A better solution here would be to run the test on P4 and expect an error. 

I can update it in a follow-up MR, it is out of scope of this MR

Reply:
Now fixed as you suggested. All the test cases are run on P4 and an error is expected for `rx_buffer` test run on P4

Reply:
Great, thank you!